### PR TITLE
Final UX improvement in PPV2

### LINKED ISSUE #28934: BO - A bad display of buttons in combinations tab with product page V2 when multistore enabled
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

A bad display of buttons in combinations tab with product page V2 when multistore enabled
image

image

### Expected behavior

The buttons are well displayed

### Steps to reproduce

1. Enable multistore and create a new shop
2. BO > Advanced Parameters > New & Experimental Features > Enable new product page 
3. Create a product with combinations
4. Click to generate a new combinations
5. See error >> the buttons (edit, delete, radio) are bad displayed

### PrestaShop version(s) where the bug happened

8.0.x, 8.0.0

### PHP version(s) where the bug happened

7.2

### If your bug is related to a module, specify its name and its version

_No response_

### LINKED ISSUE #31417: PPV2 - The number of filter is not well displayed when filtering more than one in a product with combinations
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

The number of filters is not well displayed when filtering more than one in a product with combinations 

**PS8.1.0_beta_build1 : NOK** :x: 

image

**PS8.1.0_beta_build2 : NOK** :x: 

image

https://user-images.githubusercontent.com/92912932/219618605-947c58c0-701e-4202-93e0-d58e25d27897.mp4

**PS8.0.1 : OK** :heavy_check_mark: 

image

https://user-images.githubusercontent.com/92912932/219619437-e880c75a-a1ca-47c1-b738-f3ac73338342.mp4

So, it's a regression on **8.1.0_beta**!

### Expected behavior

The number of filters is well displayed

### Steps to reproduce

1. Use the 8.1.0_beta
2. Go to BO > Advanced parameters > New & Experimental features > Enable New product page
3. Go to BO > Catalog > Products V2 > Create a product with combinations
4. Add new combinations (eg: more than 2 combinations) for the product 
5. Click on Save
6. On the `Manage product combinations` section > Use Filter by > Choose the first filter
7. Click on the second filter 
8. See error >> The filter number is not well displayed
9. Click on the third/fourth.. filter 
8. See error >> The filter number is not well displayed

### PrestaShop version(s) where the bug happened

8.1.0_beta

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

_No response_

### LINKED ISSUE #31424: PPV2 : The filter number is not displayed at all with a product with a combination when filtering from Right to left
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

The filter number is not displayed at all with a product with a combination when filtering from Right to left. 
Only **Clear filter** is displayed.

**PS8.1.0_beta_build1 : NOK** :x: 
image

**PS8.1.0_beta_build2 : NOK** :x: 
image

https://user-images.githubusercontent.com/92912932/219678934-747b4bc8-bd65-4bdb-bc74-d3c04c643ce1.mp4

**PS8.0.1 : OK** :heavy_check_mark: 

image

https://user-images.githubusercontent.com/92912932/219678868-ec119ed5-e837-491d-9309-68c7ee30fe38.mp4

So, it's a regression on **8.1.0_beta**!

### Expected behavior

The number of filtered cases is displayed!

### Steps to reproduce

1. Use the 8.1.0_beta
2. Go to BO > Advanced parameters > New & Experimental features > Enable New product page
3. Go to BO > Catalog > Products V2 > Create a product with combinations
4. Add new combinations (eg: more than 2 combinations) for the product
5. Click on Save
6. On the Manage product combinations section > Use Filter by > Choose the last filter **from right to left**
7. Click on the next filter
8. See error >> Only Clear filter is displayed and the number is not displayed

### PrestaShop version(s) where the bug happened

8.1.0_beta

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

_No response_

### LINKED ISSUE #31588: Combination improvement on the new product page
### Combination list

- [x]  Remove the reset mode because it is redundant with the cancel mode which causes exactly the same number of clicks (already done I believe)
- [x]  Display the edition mode at first click, and not at the blur (it was done on change, not on first click no reason to turn the edition mode until something has actually been edited no?)
- [x] On the edition mode, out-of-scope parameter can be hidden (need to search what else should be hidden/disabled: footer, tabs, ... To be experimented)

### Combination generation

- [x]  Close the modal after generation

image

Acceptance test:When I click on generate combination, the modal close itself at the ending of the generation

### Combination modal

- [x]  Hide the history panel by default and displays it only after at least 2 pages edit

- [x] Adapt the Cancel button

When the modal has been edited
The button to close the modal is called 'Cancel'

When the modal has not been edited
The button to close the modal is called 'Close '

### Additional checks

- [x] https://github.com/PrestaShop/PrestaShop/issues/31660
- [x] https://github.com/PrestaShop/PrestaShop/issues/28934
- [x] https://github.com/PrestaShop/PrestaShop/issues/31417
- [x] https://github.com/PrestaShop/PrestaShop/issues/31424

### LINKED ISSUE #31589: UI mobile on the new product page
## Mockups:
image

- Apply modification of display for the header based on the mockups
- We hide the references in the header in mobile
- In mobile don't make for now the header sticky
- The actions present next to the tabs (sales, product list, ...) should be accessed in the header (see if it implies having two sets of buttons that switch, or if it should always be in the header)
- put the all the buttons in the dropdown in the footer in desktop and mobile view except for 'go to catalog', 'cancel' and 'save and published' (should be most handled for desktop mode already)
- Make the cancel button as an icon in mobile view
- make the tab sticky in the header if possible (NTH)

## Combination modal

- Hide the 'Edited combination'
- The button 'Next combination >' become '>'
- The button '< Preview combination' become '<'

## Combination List

- Combinations list has a Horizontal scroll bar 
- Min size for the column 

## Footer

The footer has too many interactive buttons (most buttons should be in dropdown)

### Acceptance test
When the product page is displayed on mobile without any modification, the footer only contains:
- go to catalog
- Save and publish
- 3 dot button (containing: add new product, delete, duplicate and Preview)

When the product page is displayed on mobile with one modification, the footer only contains:
- Cancel 
- Save and publish

### LINKED ISSUE #31660: Product V2 - Bad display of edit combination modal only with language with accent (FR/IT)
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

- The qty to add/substruct input is displayed with height bigger than the standard
- ` &nbsp;` is displayed between ***Edit combination** and **:**

image

### Expected behavior

- The qty to add/substruct input should displayed with standard height
- ` &nbsp;` shouldn't be displayed between ***Edit combination** and **:**

### Steps to reproduce

1. Change your admin language to French or Italian language
2. CONFIGURE>Advanced Parameters>New & Experimental Features
3. Enable **New product page - Single store** option
4. Save
5. SELL>Catalog>Products
6. Edit a combination
7. The modal of edit combination is displayed
8. Check the qty input => :x: See error the input is so much bigger than usual
9. Change the qty by adding some 
10. Save
11. Check the combinations you have edited, the **Edited combinations** block :x: 
12. See error =>` &nbsp;` is displayed between ***Edit combination** and **:**

### PrestaShop version(s) where the bug happened

8.0.0, 8.0.1, 8.0.2 and 8.1.0

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | This PR fixes a fe UX bugs and add a few improvements as well on combination elements: the generator, the edition modal (and its history panel), the edition mode<br>It also bring a few fixes and improvements on the whole form in mobile view
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | See the issue that summarizes what had to be improved for UX, and a few other issues are (or were already) fixed as well, they should be checked so we can close them
| Fixed ticket?     | Fixes #31588 Fixes #31424 Fixes #31417 Fixes #28934 Fixes #31660 Fixes #31589
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Comment by jolelievre** [comment]
Running auto campaign https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4560090225

**Review by 0x346e3730 — DISMISSED** [review]

**Review by kpodemski — DISMISSED** [review]

File: tests/UI/campaigns/productV2/functional/03_CRUDProductWithCombinations.ts [hunk 1]
```diff
@@ -243,11 +243,11 @@ describe('BO - Catalog - Products : CRUD product with combinations', async () =>
...
 
-    it('should close generate combinations modal', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'closeGenerateCombinationsModal', baseContext);
+    it('combinations generation modal should be closed', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'generateCombinationsModalIsClosed2', baseContext);
 
-      const isModalVisible = await combinationsTab.closeGenerateCombinationModal(page);
-      await expect(isModalVisible).to.be.true;
+      const isModalClosed = await combinationsTab.generateCombinationModalIsClosed(page);
+      // await expect(isModalClosed).to.be.true;
```
Comment (zuk3975) [inline tests/UI/campaigns/productV2/functional/03_CRUDProductWithCombinations.ts]: is it commented for a reason?

Comment (jolelievre) [reply tests/UI/campaigns/productV2/functional/03_CRUDProductWithCombinations.ts]: It's a temporary fix, I don't know why it doesn't work in the CI but locally I have no issues, so I was testing things But there's another problem anyway, I need to look more into this

**Review by djoelleuch — DISMISSED** [review]
Hello @jolelievre 
The automated tests are OK. (https://github.com/djoelleuch/ga.tests.ui.pr/actions/runs/4596815477/jobs/8118756180) :heavy_check_mark: 
image
image

[Untitled_ Apr 3, 2023 3_28 PM.webm](https://user-images.githubusercontent.com/115152460/229543161-829cfcb4-9303-42fc-8aa6-9d35dd476cbf.webm)

It is QA :heavy_check_mark: 
Thank you :rocket:

**Comment by jolelievre** [comment]
Running campaign https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4602722352

**Comment by jolelievre** [comment]
Running campaign https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4637569771

**Review by zuk3975 — DISMISSED** [review]

**Comment by jolelievre** [comment]
Running campaign https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4650715508

**Comment by jolelievre** [comment]
Running campaign https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4670316298

**Review by zuk3975 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by djoelleuch — APPROVED** [review]
Hello @jolelievre ,
LGTM , The issues are well fixed , It's QA approved :heavy_check_mark: 
Automated tests --> OK

Thank you for you PR

**Review by Progi1984 — APPROVED** [review]
Ok for UI Tests

**Review by nesrineabdmouleh — APPROVED** [review]

**Pull request merged**